### DISCUSSION TITLE
JMeter timing out in communicating to InfluxDB2.7.7 Container instance

### DISCUSSION THREAD

**Discussion by Vamsi8591** [mderevyankoaqa/jmeter-influxdb2-listener-plugin#106] [Q&A]
Hi all,

       I got below error when implemented "jmeter-influxdb2-listener-plugin"
       -----------------------------------------------------------------------------------------
2024-08-16 13:47:46,389 ERROR i.g.m.i.v.InfluxDatabaseBackendListenerClient: Error has occurred, batch with size 4 was not imported, see the details --> Connect timed out

InfluxDb2.7.7 Container instance log:-
--------------------------------------------------
2024-08-16 13:54:50 ts=2024-08-16T08:24:50.186563Z lvl=error msg="Unable to write gathered points" log_id=0r2ZIpSl000 service=scraper scraper-name="new target" error="database not found: 0a1647670a6ae644
<img width="959" alt="image (1)" src="https://github.com/user-attachments/assets/1b5dd54a-0d37-46a5-a337-36defbf94921">

 
 Note:- URL used "http://ContainerIP:8086
----------------------------------------------------------------------------------------

**Comment by Laharigopa** [comment]
Adding to above issue, updated with screenshot
<img width="704" alt="Jmeter_Influx" src="https://github.com/user-attachments/assets/f637adba-9680-4f5b-8d48-2ff36cf0ee98">



We tested grafana and influxdb container instances connectivity but jmeter is timing out when sending metrics to influxdb container instances when used this plugin in backend listener.
